handlers.go

```diff
@@ -781,7 +781,7 @@ func (s *Server) handleAddCar(c echo.Context, u *User) error {
 
 	go func() {
 		// TODO: we should probably have a queue to throw these in instead of putting them out in goroutines...
-		s.CM.ToCheck <- cont.ID
+		s.CM.toCheck(cont.ID)
```

Comment:
I think we should name this `checkContent` or maybe `check` if this is going to be a function

Reply:
these will come in part 2